public/components/datasources/components/manage/integrations/installed_integrations_table.tsx

```diff
@@ -148,9 +149,25 @@ export const InstallIntegrationFlyout = ({
     ),
   };
 
+  const [installingIntegration, setInstallingIntegration] = useState<string | null>(null);
```

Comment:
To allow installing in a flyout: We add the optional parameter `setInstallingIntegration` that is used to conditionally override the href to the setup page with an onClick that sets `installingIntegration` to the selected integration. (See diff in `available_integration_table.tsx`.)

Reply:
nit: if we switch to use `navigateToApp()`, do we still need this?

Reference:
Implementation of the EuiLink for datasource: https://github.com/opensearch-project/dashboards-observability/blob/31d506a1672803060d6abb7699ae099fbeb0b628/public/components/datasources/components/manage/accelerations/flyout_modules/acceleration_details_tab.tsx#L107